getting-started-5.html

```diff
@@ -43,7 +43,7 @@
 
 <li>
 <p class="first">Download the OMERO.insight client at:
-<a href="http://downloads.openmicroscopy.org/latest/omero5" target="_blank" class="external-url">http://downloads.openmicroscopy.org/latest/omero5</a></p>
+<a href="https://downloads.openmicroscopy.org/latest/omero5" target="_blank" class="external-url">https://downloads.openmicroscopy.org/latest/omero5</a></p>
```

Comment:
Again, we probably want to replace this with the /omero/downloads link to the new website